protocol/dubbo/dubbo_invoker.go

```diff
@@ -47,6 +47,7 @@ var (
 
 // DubboInvoker ...
 type DubboInvoker struct {
+	sync.RWMutex
```

Comment:
u should add a lock name.

Reply:
May be using a `isDestroyed` status is good.